seedcase_sprout/core/edit_package_properties.py

```
@@ -43,16 +40,27 @@ def edit_package_properties(path: Path, properties: dict) -> dict:
 
     Raises:
         FileNotFound: If the `datapackage.json` file doesn't exist.
-        NotPropertiesError: If the new package properties are not correct or the current
-            package properties are not well-formed.
         JSONDecodeError: If the `datapackage.json` file couldn't be read.
+        ExceptionGroup: If there is an error in the current, incoming or resulting
+            package properties. A group of `CheckError`s, one error for each failed
+            check.
     """
     check_is_file(path)
-    verify_properties_are_well_formed(properties, PackageError.type)
+
+    properties.pop("resources", None)
```

This just removes `resources` from the incoming properties object. I added this because this function is about updating package properties, so we don't want to overwrite or update resource properties.
Maybe it can be argued that it would make sense to allow setting `resources` to `[]`, for example, but that could also be achieved by deleting all resources.

But I'm open to doing it in a different way! E.g., we could throw an error if `resources` are set.